sklearn/mixture/tests/test_gmm.py

```diff
@@ -327,6 +327,26 @@ def test_1d_1component():
         assert_array_almost_equal(g.bic(X), g_full_bic)
 
 
+def test_fit_predict():
+    """
+    test that gmm.fit_predict works as expected
+    """
```

Comment:
It's better to set a local `rng` instance inside the test here to make the test outcome independent of the execution order:

```python
def test_fit_predict():
    rng = np.random.RandomState(0)
     # ...
```